docs/product-user-guide/src/main/asciidoc/test-scenarios-create-proc.adoc

```diff
@@ -1,88 +1,55 @@
-[id='test_scenarios_create_proc']
-= Creating and Running a Test Scenario
+[id='test-scenarios-create-proc']
+= Creating and running a test scenario
 
-A basic test scenario requires, at minimum, the following data:
+You can create test scenarios in {CENTRAL} to test the functionality of business rule data before deployment. A basic test scenario must have at least the following data:
 
 * Related data objects
 * *GIVEN* facts
-* *EXPECT* facts
```

Comment:
On the same theme, but this looks strange. GIVEN, EXPECT. Consider Related data objects, Given facts, Expected results?